src/fswAlgorithms/attGuidance/prescribedRot2DOF/prescribedRot2DOF.h

```diff
@@ -21,13 +21,40 @@
 
 /*! Include the required files. */
 #include <stdint.h>
+#include <stdbool.h>
 #include "architecture/utilities/bskLogging.h"
 #include "cMsgCInterface/HingedRigidBodyMsg_C.h"
 #include "cMsgCInterface/PrescribedMotionMsg_C.h"
 
 /*! @brief Top level structure for the sub-module routines. */
 typedef struct
 {
+    /* User configurable variables */
+    double phiDDotMax;                                         //!< [rad/s^2] Maximum angular acceleration of the spinning body
+    double rotAxis1_M[3];                                      //!< M frame rotation axis for the first rotation
+    double rotAxis2_F1[3];                                     //!< F1 frame intermediate rotation axis for the second rotation
+
+    /* Private variables */
+    double r_FM_M[3];                                          //!< [m] Position of the F frame origin relative to the M frame origin in M frame components
+    double rPrime_FM_M[3];                                     //!< [m/s] B frame time derivative of r_FM_M in M frame components
+    double rPrimePrime_FM_M[3];                                //!< [m/s^2] B frame time derivative of rPrime_FM_M in M frame components
+    double omega_FM_F[3];                                      //!< [rad/s] angular velocity of frame F relative to frame M in F frame components
+    double omegaPrime_FM_F[3];                                 //!< [rad/s^2] B frame time derivative of omega_FB_F in F frame components
+    double sigma_FM[3];                                        //!< MRP attitude of frame F relative to frame M
+    bool convergence;                                          //!< Boolean variable is true when the attitude maneuver is complete
```

Comment:
:small_red_triangle_down: What do you think about renaming this to something like `isManeuverComplete`? Or just something that indicates what has converged.

Reply:
Yes, I made that change!